### ISSUE TITLE
It is impossible to find  current line and column number when using a screen reader.

### ISSUE THREAD

**Issue by jareds** [microsoft/vscode#78220]
- VSCode Version: 1.36.3
- OS Version: Windows 10 insider

Steps to Reproduce:

1. Open any file in the edotor.

Does this issue occur when all extensions are disabled?: yes

It is impossible to get the current line and column number when using screen reading software. This is a particular issue when editing files where indentation matters such as YAML. For example

```
item1:
  item2:
    - value
    item3:
      - value
  item2a
```

To know that item2a is at the same level as item2 I need to arrow over from the beginning of the text and count spaces until I encounter a new line. The deeper the indentation level gets the more difficult this is to do.

**Comment by zersiax** [comment]
I will add here that the go to line dialog will tell you your current line. However, a quick way to query the current column is indeed missing, at least to my knowledge.

**Comment by Neurrone** [comment]
Hm, I think this is only an issue if not using the NVDA screen reader, since it has excellent facilities for reporting indentation comparable to the way a sighted person would see it, such that this functionality isn't needed, even for the largest of YAML documents.

This could be an issue if using jaws though.

**Comment by isidorn** [comment]
We could change the Go to Line command, such that it also announces the current column.
"Current Line: 26, Column: 12. Type a line number..."
Let me know what you think about that and we can easily add it.

Code pointer
https://github.com/Microsoft/vscode/blob/7c5e0803df31f485243eb5e6629862d873e9540f/src/vs/workbench/contrib/quickopen/browser/gotoLineHandler.ts#L104

**Comment by travisroth** [comment]
Thank you for looking into this. My response: opening a dialog to get this information is to slow. I'd prefer the cursor location be printed on the status line. My understanding is VSCode does do this when not in accessibility/screen reader mode. At present, VSCode shows "screen reader mode" message on the status line when in accessibility mode. This message is not useful. Yes when I am using a screen reader I am absolutely aware I am using one. Screen readers have commands to read the status line of editors already built in, so if this information is present it can be efficiently accessed. Note: at present JAWS is not reading the status line effectively, which may be another issue. VSCode's status line area may need a role=status assigned, and then JAWS may need a script added, but if the information is displayed we have a much better chance of getting screen readers to access it.
JAWS has another column and line orientation checking command: alt+numpad del. This command does return the column of the cursor, and appears to be accurate although this should be tested more. However the line number is always 1. This is true in other Electron apps too, so this may be something between Electron, UIAutomation and JAWS. But is another possible solution. Although I'm guessing fixing the status line is easier, and will be supported by more screen readers. NVDA is quite popular. However, JAWS is still a dominant force in the professional/employed ranks of screen reader users.

**Comment by jareds** [comment]
I like the idea of having go to line announce column number as a short term fix. Ideally it would be possible to heavily customize the way information is announced, for example regardless of screen reader announce the current column number. I realize people's opinions on how things should be announced will very quite a bit. This is why I would like to see
https://github.com/microsoft/vscode/issues/66091
get implemented. It would then be possible to create an extension that could provide information to screen readers such as line and column number with out requiring a dialog to open.

**Comment by jareds** [comment]
@isidorn Do you want me to create a pull request for this? I've been able to get line and column announced by changing the getAriaLabel method.

**Comment by isidorn** [comment]
@jareds a PR is welcome. Though I think we can just change the text fully, not only the aria label. So that all users see the column. I provided the code pointer in my comment from above.

**Comment by isidorn** [comment]
I have added the colum information to the Go To Line command.
Verifier: trigger Go To Line command and make sure the Column is seen and read out by the screen reader.
